creusot-contracts/src/std/ptr.rs

```diff
@@ -1,6 +1,16 @@
 use crate::*;
 pub use ::std::ptr::*;
 
+/// Integer representation of a pointer in logic.
+///
+/// This returns `u64` because there is no support for builtins with architecture-dependent types.
+/// In any case it is enough to support both 32-bit and 64-bit architectures.
```

Comment:
An easy way to use `usize` instead of `u64` here is to define in the prelude three different versions of `creusot.prelude.Opaque.addr_logic`, for the three different bit widths, and then use `cfg` to chose the right one.

See, for example what we are doing for `[T]::view` in `slice.rs`.

Reply:
Thanks!